### ISSUE TITLE
Not work with jQuery 1.11.0

### ISSUE THREAD

**Issue by phpclub** [Studio-42/elRTE#12]
Cannot read property 'msie' of undefined 
http://api.jquery.com/jquery.browser/

**Comment by phpclub** [comment]
Fix by http://blog.jquery.com/2013/05/08/jquery-migrate-1-2-1-released/ 
/jquery-migrate

**Comment by barma** [comment]
dependency on another library - this is a bad decision

**Comment by baconbrad** [comment]
Not really, elRTE depends on both jQuery and jQuery UI. It saved the developers so much time in developing what these libs provide. Migrate simply adds backward support for depreciated jQ features. It is not some third party lib nor does it add any extra bulk (since it just adds the feature removed anyways.) Importantly it allows end user developers to have the ability to use a much wider range of jQ versions. Since some might be restricted to certain libs. So increasing the jQ version requirement by doing a core change would actually be a bad move in itself. At any rate, tons of web sites including some of the most massive websites use Migrate without issue.
